plat/allwinner/common/plat_helpers.S

```diff
@@ -9,29 +9,10 @@
 #include <sunxi_def.h>
 #include <sunxi_mmap.h>
 
-	.globl	plat_crash_console_init
-	.globl	plat_crash_console_putc
-	.globl	plat_crash_console_flush
 	.globl	plat_my_core_pos
 	.globl	platform_mem_init
 	.globl	plat_report_exception
```

Comment:
If you remove this, you won't get crash messages anywhere, your multi console isn't setup to print crash messages. The default implementation doesn't do anything (check ``plat/common/aarch64/platform_helpers.S``).